sphinx_gallery/components/sg_download_links.html

```diff
@@ -0,0 +1,12 @@
+{# Displays the download link for the Python source code of the example. #}
+{% set download_links = get_download_links() %}
+{% for key, link_info in download_links.items() %}
+  {% if link_info %}
+    <div class="sphx-glr-download-{{ key }}-sidebar" title="{{ link_info.title }}">
```

Comment:
Like maybe 
```suggestion
    <div class="sphx-glr-download-sidebar sphx-glr-download-{{ key }}-sidebar" title="{{ link_info.title }}">
```
then images inside divs with class `sphx-glr-download-sidebar` can have `max-height: 20px` by default in SG's CSS?